client/src/main/kotlin/io/spine/examples/shareaware/client/Theme.kt

```diff
@@ -150,4 +155,5 @@ private object Colors {
     val DarkGrey = Color(0xff5b595f)
     val Red = Color(0xffff3b30)
     val Scrim = Color.Black.copy(alpha = 0.5f)
+    val Green = Color(52, 199, 89)
```

Comment:
Consider using the same color notation in all places, such as HEX(0xaarrggbb) as in the colors above.